### DISCUSSION TITLE
N64 controller connector pins too large for holes on PCB.

### DISCUSSION THREAD

**Discussion by JCreazy** [sanni/cartreader#392] [Need help]
As you can see.
![PXL_20220625_033820701 MP](https://user-images.githubusercontent.com/34291350/175756861-42c0ea58-8cd0-4712-b174-271569893d98.jpg)

**Comment by sanni** [comment]
They are smaller so that people don't accidentally solder the connector directly to the PCB. Because there is no space since the N64 cartridge goes there.

The plan is to add a bracket for the connector to the frame/case and then solder a wire from the connector to the PCB. I have not decided yet where to best put it though.

**Reply by JCreazy** [reply]
Would it be possible to move the connector closer to the edge to avoid interferring with the N64 cartride or perhaps moved to the opposide side of the cartridge?

**Comment by sanni** [comment]
There is no space left on the PCB that is not covered by a cartridge.

![IMG_1349](https://user-images.githubusercontent.com/19169494/176994148-5406316e-bed4-40b8-96de-06bb837a39dc.jpg)

![IMG_1350](https://user-images.githubusercontent.com/19169494/176994150-2862516c-cfc1-44eb-b3df-4d79371fd366.jpg)
